pkg/types/azure/defaults/platform.go

```diff
@@ -4,12 +4,20 @@ import (
 	"github.com/openshift/installer/pkg/types/azure"
 )
 
+var (
+	// Overrides
+	defaultMachineClass = map[string]string{}
+)
+
 // SetPlatformDefaults sets the defaults for the platform.
 func SetPlatformDefaults(p *azure.Platform) {
 }
 
-// InstanceClass returns the instance "class" we should use for a given
-// region.
+// InstanceClass returns the instance "class" we should use for a given region.
 func InstanceClass(region string) string {
```

Comment:
This function should now be private for this package, right?

Reply:
As we have the `<role>MachineType` functions.

Reply:
I have renamed the function and made it private.